lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -1173,7 +1173,13 @@ class DefaultAssetPickerBuilderDelegate
             ),
           ),
         ),
-        const Align(alignment: AlignmentDirectional.bottomEnd, child: Icon(Icons.audiotrack)),
+        const Align(
+          alignment: AlignmentDirectional.bottomEnd,
+          child: Padding(
+            padding: EdgeInsetsDirectional.only(end: 4, bottom: 8),
```

Comment:
Could we do this by setting the `alignment` property?

Reply:
If I use `alignment`, will RTL still be supported here?

Reply:
`AlignmentDirectional` supports RTL.

Reply:
No good, using `Alignment` breaks support for RTL